xml/System.Windows.Markup/TypeExtension.xml

```diff
@@ -294,7 +294,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the type name represented by this markup extension.</summary>
-        <value>A string that identifies the type. This string uses the format *prefix*<c>:</c>*className*. (*prefix* is the mapping prefix for an XML namespace, and is only required to reference types that are not mapped to the default XML namespace for WPF ([!INCLUDE[TLA#tla_wpfxmlnsv1](~/includes/tlasharptla-wpfxmlnsv1-md.md)]).</value>
+        <value>A string that identifies the type. This string uses the format *prefix*<c>:</c>*className*. (*prefix* is the mapping prefix for an XML namespace, and is only required to reference types that are not mapped to the default XML namespace for WPF http://schemas.microsoft.com/winfx/2006/xaml/presentation.</value>
```

Comment:
```suggestion
        <value>A string that identifies the type. This string uses the format *prefix*<c>:</c>*className*. (*prefix* is the mapping prefix for an XML namespace and is only required to reference types that are not mapped to the default XML namespace for WPF (http://schemas.microsoft.com/winfx/2006/xaml/presentation).</value>
```

Also should it be code fenced?

Reply:
It should. I wanted to test whether it would render a link automatically or not. I'll make changes to your suggestion.